react/SearchWrapper.tsx

```diff
@@ -331,9 +337,18 @@ const SearchWrapper: FC<SearchWrapperProps> = props => {
 
   const canonicalLink = useCanonicalLink()
 
+  const helmetCanonicalLink = getHelmetLink({
+    canonicalLink,
+    page: pageFromQuery,
+    map: canonicalWithoutUrlParams ? '' : map,
+    term: termFromQuery,
+    rel: 'canonical',
```

Comment:
I've noticed that if you add any params other than `_q` or `map` it will not be returned in the `og:url` tag. Is this a problem?

https://arthur--budgetgolf.myvtex.com/black%20shoes?_q=black%20shoes&map=ft&order=OrderByBestDiscountDESC

Reply:
I think that's best practice for Google. Sort and filter variations should share the same canonical as the standard search.